Makefile

```diff
@@ -2,6 +2,7 @@
 
 bootstrap:
 	./scripts/download-readmes.js
+	node ./scripts/download-sponsors.js
```

Comment:
I think you can just put `#!/usr/bin/env node` at the top like in the file above and you don't need the `node` part

Reply:
Yep + `chmod +x scripts/download-sponsors.js`